webui/react/src/components/DatePickerFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import { DatePicker } from 'antd';
+import { Moment } from 'moment';
+import React from 'react';
+
+import css from './DatePickerFilter.module.scss';
+import Label from './Label';
+
+interface Props {
+  label: string;
+  onChange?: (date: Moment | null) => void;
+  value?: Moment;
+}
+
+const DatePickerFilter: React.FC<Props> = ({ label, onChange, value }: Props) => {
+  return (
+    <div className={css.wrapper}>
+      <Label>{label}</Label>
+      <DatePicker
+        allowClear={false}
+        showNow={false}
+        showTime
+        style={{ width: '130px' }}
```

Comment:
**nit:** I think this might not be needed anymore since it's shown inside a narrow dropdown. Can/should datetime picker span to show time as well?